satpy/etc/readers/viirs_sdr.yaml

```diff
@@ -410,132 +445,152 @@ datasets:
+  #   file_group: VIIRS-M12-SDR
+  # svm13:
+  #   file_reader: !!python/name:satpy.readers.viirs_sdr.VIIRSSDRFileHandler
+  #   file_patterns: ['SVM13_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
+  #   file_group: VIIRS-M13-SDR
+  # svm14:
+  #   file_reader: !!python/name:satpy.readers.viirs_sdr.VIIRSSDRFileHandler
+  #   file_patterns: ['SVM14_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
+  #   file_group: VIIRS-M14-SDR
+  # svm15:
+  #   file_reader: !!python/name:satpy.readers.viirs_sdr.VIIRSSDRFileHandler
+  #   file_patterns: ['SVM15_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
+  #   file_group: VIIRS-M15-SDR
+  # svm16:
+  #   file_reader: !!python/name:satpy.readers.viirs_sdr.VIIRSSDRFileHandler
+  #   file_patterns: ['SVM16_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
+  #   file_group: VIIRS-M16-SDR
+  # vdnbo:
+  #   file_reader: !!python/name:satpy.readers.viirs_sdr.VIIRSSDRFileHandler
+  #   file_patterns: ['SVDNB_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
+  #   file_group: VIIRS-DNB-SDR
+  # generic_mbands:
+  #   file_reader: !!python/name:satpy.readers.viirs_sdr.VIIRSSDRFileHandler
+  #   file_patterns: ['GMODO-SVM01-SVM02-SVM03-SVM04-SVM05-SVM06-SVM07-SVM08-SVM09-SVM10-SVM11-SVM12-SVM13-SVM14-SVM15-SVM16_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
+  #   file_group: VIIRS-MOD-GEO
+  # generic_ibands:
+  #   file_reader: !!python/name:satpy.readers.viirs_sdr.VIIRSSDRFileHandler
+  #   file_patterns: ['GIMGO-SVI01-SVI02-SVI03-SVI04-SVI05_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
+  #   file_group: VIIRS-IMG-GEO
+  # generic_dnb:
+  #   file_reader: !!python/name:satpy.readers.viirs_sdr.VIIRSSDRFileHandler
+  #   file_patterns: ['GDNBO-SVDNB_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
+  #   file_group: VIIRS-DNB-GEO
+
+  generic_file:
     file_reader: !!python/name:satpy.readers.viirs_sdr.VIIRSSDRFileHandler
-    file_patterns: ['SVDNB_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
-    file_group: VIIRS-DNB-SDR
+    file_patterns: ['{datasets}_{platform_shortname}_d{start_time:%Y%m%d_t%H%M%S%f}_e{end_time:%H%M%S%f}_b{orbit:5d}_c{creation_time:%Y%m%d%H%M%S%f}_{source}.h5']
```

Comment:
Could you add a comment with a couple example filenames for this?

Reply:
Done